compiler/optimizer/InductionVariable.cpp

```diff
@@ -3450,6 +3462,7 @@ bool TR_LoopStrider::reassociateAndHoistComputations(TR::Block *loopInvariantBlo
             //node->getSymbolReference()->getSymbol()->isAuto() &&
             node->getSymbolReference()->getSymbol()->isAutoOrParm() &&
             _neverWritten->get(node->getSymbolReference()->getReferenceNumber()))) &&
+           !originalNode->getFirstChild()->isDataAddrPointer() && // Disable storing dataAddr in temps OMR/#7560
```

Comment:
Rather than referring to an issue number here, may I suggest that you add some sort of TODO marker that's specific to the work needed to allow dataAddr pointers to be stored in temps?  With issue numbers, there's always a risk that where the source code resides could change, and the issue number will no longer be meaningful.

Reply:
I can change the comment and use a TODO, but on describing what's needed here I don't see it being the better solution as a) an elaborate explanation is needed and b) its not specific to this area on both sides: where to enable, and what to fix to enable. Currently CAS intrinsic issue but not rolling out others being affected and possibly more fixes needed to support that.

I can do something generic like `// TODO enable storing dataAddrPtr in temps` with the issue pointing to code and explaining reasons.

Reply:
Yes, that was the sort of thing I was looking for.  I don't think a long detailed explanation is necessary.  Thanks!